recipe/meta.yaml

```diff
@@ -21,12 +21,13 @@ requirements:
   build:
     - pkg-config  # [not win]
     - python
-    - zeromq      # [not win or (win and py35)]
+    - zeromq      # [not win]
     - libsodium   # [not win]
   run:
     - python
-    - zeromq      # [not win or (win and py35)]
+    - zeromq      # [not win]
     - libsodium   # [not win]
+    - vs2015_runtime  # [win and py35]
```

Comment:
We hadn't been including it before. I think I agree that it may have been causing issues. So, I asked if we could add it in this issue ( https://github.com/conda-forge/python-feedstock/issues/47 ). It was proposed for addition, accepted, and built as part of PR ( https://github.com/conda-forge/python-feedstock/pull/51 ). We did the same thing for Python 3.4 in PR ( https://github.com/conda-forge/python-feedstock/pull/52 ). There were some other useful Windows patches that were added too. Hopefully these changes help.

Yes, a user would get `vs2015_runtime` with Python 3.5 on `defaults`. Now we are doing the same thing.